custom/icds_reports/tests/test_inactive_dashboard_users.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+
+from __future__ import unicode_literals
+from django.test.testcases import TestCase
+from django.test.client import RequestFactory
+from django.core.urlresolvers import reverse
+from custom.icds_reports.tasks import  collect_inactive_dashboard_users
+from custom.icds_reports.models.helper import IcdsFile
+import json
+import mock
+import datetime
+import zipfile
+import csv
+class Base(TestCase):
+
+    def setUp(self):
+        self.factory = None
+        self.user = None
+        self.view = None
+        self.url = None
+        self.run_july_third_test = False
```

Comment:
what's the purpose of this `setUp`? doesn't seem like any of these are used/needed